src/Microsoft.Extensions.Logging/LoggerBuilderExtensions.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Extensions.Logging
+{
+    /// <summary>
+    /// Extension methods for setting up logging services in an <see cref="ILoggerBuilder" />.
+    /// </summary>
+    public static class LoggerBuilderExtensions
+    {
+        public static ILoggerBuilder AddConfiguration(this ILoggerBuilder builder, IConfiguration configuration)
+        {
+            builder.Services.TryAddEnumerable(ServiceDescriptor.Singleton<IConfigureOptions<LoggerFilterOptions>>(new LoggerFilterConfigureOptions(configuration)));
+            builder.Services.TryAddEnumerable(ServiceDescriptor.Singleton<IOptionsChangeTokenSource<LoggerFilterOptions>>(new ConfigurationChangeTokenSource<LoggerFilterOptions>(configuration)));
```

Comment:
Does the "TryAddEnumerable" mean that calling AddConfiguration a second time with a different configuration source is a no-op? If so, there would be quite a mix of behaviors here (IIUC):
* AddConfiguration can only be called once and further calls are ignored
* AddProvider can be called multiple times - the same provider can even be added multiple times
* SetMinimumLevel can be called multiple times and overwrites previous values

Not sure if it's a valid scenario to call "AddConfiguration" with different configuration sources but would it be a problem if it were allowed? The IConfigureOptions system can handle this anyway, right?